go.mod

```diff
@@ -51,6 +51,7 @@ require (
 	golang.org/x/time v0.1.0
 	google.golang.org/protobuf v1.31.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
+	rogchap.com/v8go v0.9.0
```

Comment:
Unfortunately I had to add this as a new dependency because `goja` only supports ES5+ and `ethers` library is not ES5 compatible. I tried my luck with transpilers + polyfilly but couldn't really get it working with goja.